src/cpu/flexcpu/simple_dataflow_cpu.cc

```diff
@@ -692,12 +642,29 @@ SimpleDataflowCPU::DataPort::recvTimingResp(PacketPtr pkt)
 void
 SimpleDataflowCPU::DataPort::recvReqRetry()
 {
-    receivedPushback = false;
+    assert(blockedPkt);
+    DPRINTFS(SDCPUCoreEvent, cpu, "data retrying %s\n", blockedPkt->print());
```

Comment:
Was there something wrong with letting the name of the port be used to DPRINTF before?